nailgun/entities.py

```diff
@@ -7917,6 +7917,7 @@ def read(self, entity=None, attrs=None, ignore=None, params=None):
         if not ignore:
             ignore = set()
         ignore.add('hypervisor_password')
+        ignore.add('proxy')
```

Comment:
Should we add a comment saying that this is temporary and need to be removed ones related BZ is fixed?

Reply:
Done